CHANGELOG.md

```diff
@@ -5,6 +5,27 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## v0.6.1 (Unreleased)
+
+### Changed
+
+- removed `long` package (#115)
+
+#### **arch3-core**
+
+- change the type of `gasWanted` and `gasUsed`of `TxResult` from `number` to `bigint`,
+  to match the new types returned by `@cosmjs/cosmwasm-stargate` (#115)
```

Comment:
This should be marked as a breaking change since it requires library users to handle those fields differently.